src/test/java/foodwhere/model/AddressBookTest.java

```diff
@@ -87,7 +104,6 @@ public void getStallList_modifyList_throwsUnsupportedOperationException() {
 
     @Test
     public void addStallAddReview_generalTesting_success() {
-        AddressBook addressBook = new AddressBook();
         String testName = "test stall";
```

Comment:
I think this test can also be improved to set the address of the stall and review to be the same (and perhaps checking for the cases where one differs).

Reply:
Fixed in latest commit